app/components/searchBar/index.test.tsx

```diff
@@ -0,0 +1,31 @@
+import { screen, render, fireEvent, getByTestId } from '@testing-library/react';
+import SearchBar from '.';
+
+it('renders a input field', () => {
```

Comment:
Follow setup and teardown, and organise tests `describe blocks`.
https://jestjs.io/docs/setup-teardown

Reply:
In other components, tests were written like this only, okay I'll look into that & confirm with the AO

Reply:
I agree with Tanishq. Using describe blocks, you can put multiple individual tests which are related to same user flow.

Reply:
working on it